modules/develop/pages/produce-data/leader-pinning.adoc

```diff
@@ -0,0 +1,39 @@
+= Leader Pinning
+:page-categories: Clients, Development
+// tag::single-source[]
+:description: Learn about leader pinning and how to configure a preferred partition leader based on availability zones.
+
+Produce requests to Redpanda topics go through the topic partition leader, which then ensure that messages are synced across its follower replicas. For a Redpanda cluster deployed across multiple availability zones (AZs), leader pinning is a mechanism to ensure that a topic's partition leaders are geographically closer to clients. Similar to xref:develop:consume-data/follower-fetching.adoc[follower fetching] for consume requests, leader pinning can lower networking costs and help guarantee lower latency by routing produce requests to brokers located in certain AZs.
```

Comment:
> leader pinning can lower networking costs and help guarantee lower latency by routing produce requests

This also applies to consume requests without follower fetching, maybe we should mention that to avoid confusion.